pages/box_buster.vue

```diff
@@ -40,6 +40,8 @@
         <span>{{ total_with_maps }} plates with plates maps,</span>
         <span>{{ total_without_maps }} without.</span>
         <span>Total {{ total_positives }} positives.</span>
+        <br />
+        <span>Box barcodes scanned: {{ barcodes_scanned.join(', ') }}</span>
```

Comment:
I'm wondering if this should either be hidden initially or start by showing `0`.

Reply:
you can't see any of the box summary until the first barcode has been scanned so this shouldnt be an issue